components/admin_console/billing/payment_info_edit.tsx

```diff
@@ -1,17 +1,77 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import {FormattedMessage} from 'react-intl';
+import {useDispatch, useSelector} from 'react-redux';
 
+import {loadStripe} from '@stripe/stripe-js';
+import {Elements} from '@stripe/react-stripe-js';
+
+import {getCloudCustomer} from 'mattermost-redux/actions/cloud';
+import {getConfig} from 'mattermost-redux/selectors/entities/general';
+
+import {completeStripeAddPaymentMethod} from 'actions/cloud';
 import BlockableLink from 'components/admin_console/blockable_link';
+import FormattedMarkdownMessage from 'components/formatted_markdown_message';
+import PaymentForm from 'components/payment_form/payment_form';
 import SaveButton from 'components/save_button';
+import {areBillingDetailsValid, BillingDetails} from 'types/cloud/sku';
+import {GlobalState} from 'types/store';
+import {browserHistory} from 'utils/browser_history';
 
-type Props = {
+import './payment_info_edit.scss';
+import AlertBanner from 'components/alert_banner';
 
-};
+const STRIPE_CSS_SRC = 'https://fonts.googleapis.com/css?family=Open+Sans:400,400i,600,600i&display=swap';
+const STRIPE_PUBLIC_KEY = 'pk_test_ttEpW6dCHksKyfAFzh6MvgBj';
```

Comment:
Given that we need this in two places we should move it to the `payment_form/stripe.ts` file so we just need to update it one place there for production

Reply:
curious as to where are we getting this key from? I would refrain from using this key altogether and have it be an ENV variable. High risk if this is our key and we're sharing with the world.

Reply:
This is not our key, is a public Stripe test key. Our key is not on the public repo and is replaced only for production build.

Reply:
👍